cli/pkg/resourcemanager/delete.go

```diff
@@ -4,15 +4,12 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"strings"
 )
 
 const VerbDelete Verb = "delete"
 
 func (c client) Delete(ctx context.Context, id string, format Format) (string, error) {
-	if c.deleteSuccessMsg == "" {
```

Comment:
each command support is decided by the server. we try to send the request, and if the method is not enabled, we get a `405 method not allowed`. This also works for any methods now. This effectively moves and encapsulates the knowledge of what verb supports each resource on the server